08.ls_object/lib/ls_command.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative 'ls_params'
+require_relative 'ls_long_formatter'
+require_relative 'ls_short_formatter'
+require_relative 'ls_file_outputter'
+
+ls_params = LsParams.new(ARGV)
+
+def run(ls_params)
+  option = ls_params.option
+  path = ls_params.path
+
+  if !File.exist?(path)
+    puts "#{path}: No such file or directory"
+  else
+    return if Dir.empty?(path)
```

Comment:
これってこういうことじゃないでしょうか？
```suggestion
  elsif !Dir.empty?(path)
```

Reply:
そういうことでした・・・
ありがとうございます🙇🏻‍♀️